js/VES_0.0.1.js

```diff
@@ -4503,7 +4503,6 @@ function setReviewModal(data){
 							<div class="question-wrapper">
 								<h4 class="question"> How many hours per week do you devote to this course? </h4>
 								<h4 class="response">${results["hoursPerWeek"]}</h4>
-								<input type="range" min="0" max="50" value=${results["hoursPerWeek"]} class="p_rate disabled" disabled><br/>
```

Comment:
this is also not part of the organization question - removed sliding bar from hours per requested